reference/platform-app-yaml.md

```diff
@@ -163,6 +163,23 @@ application in MB.
 > The minimal recommended disk size is 256MB. If you see the error **UserError: Error building the project: Disk size may not be smaller than 128MB**, increase the size to 256MB.
 
 
+## Runtime
+
+The `runtime` defines sizing hints (in MB) used to determine the maximum amount of requests to process concurrently (i.e. number of workers) for a PHP instance depending on its size. Those are not hard limits.
+
+Example:
```

Comment:
Yeah, we need to add a sentence here that says: "The more memory your application consumes, the less workers it can have at a particular service size."